### ISSUE TITLE
result

### ISSUE THREAD

**Issue by fangxinGod** [henrryzh1/UR-DMU#15]
I would like to ask, can the auc in the txt document generated after the final training be used as the judgment standard for the entire result?

**Comment by henrryzh1** [comment]
Certainly, but it's best to choose your checkpoint in conjunction with the loss logs and other metrics.
